CONTRIBUTING.md

```diff
@@ -0,0 +1,139 @@
+<!-- omit in toc -->
+# Contributing to Unified Planning Library
+
+First off, thanks for taking the time to contribute! ❤️
+
+All types of contributions are encouraged and valued. See the [Table of Contents](#table-of-contents) for different ways to help and details about how this project handles them. Please make sure to read the relevant section before making your contribution. It will make it a lot easier for us maintainers and smooth out the experience for all involved. The community looks forward to your contributions. 🎉
+
+> And if you like the project, but just don't have time to contribute, that's fine. There are other easy ways to support the project and show your appreciation, which we would also be very happy about:
+> - Star the project
+> - Tweet about it
```

Comment:
Do we want to refer to twitter here? Not sure whether I would like to support it. What about "post on it on social networks"?